docs/cudf/source/dask-xgb-10min.ipynb

```diff
@@ -92,75 +134,113 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
     "size = 1000000\n",
     "npartitions = 8\n",
     "\n",
-    "pdf = pd.DataFrame({'x': np.random.randint(0, npartitions, size=size), 'y': np.random.normal(size=size)})\n",
-    "pdf = dask.dataframe.from_pandas(pdf, npartitions=npartitions)\n",
-    "\n",
-    "ddf = dask_cudf.from_dask_dataframe(pdf)\n",
+    "cdf = cudf.DataFrame({'x': np.random.randint(0, npartitions, size=size), 'y': np.random.normal(size=size)})\n",
```

Comment:
We can use `cupy.random.randint` & `cupy.random.normal` to avoid H2D copies.

Reply:
perfect. Will update.